app/code/Magento/CatalogImportExport/Model/Import/Product/Option.php

```diff
@@ -1625,14 +1636,15 @@ private function restoreOriginalOptionTypeIds(array &$typeValues, array &$typePr
     }
 
     /**
-     * Identify ID of the provided option type by its title in the specified store.
+     * Identify ID of the provided option type by its title or sku in the specified store.
      *
      * @param int $optionId
      * @param int $storeId
      * @param string $optionTypeTitle
+     * @param string $optionSku
      * @return int|null
      */
-    private function getExistingOptionTypeId($optionId, $storeId, $optionTypeTitle)
+    private function getExistingOptionTypeId($optionId, $storeId, $optionTypeTitle, $optionSku)
```

Comment:
When applicable, I would assign types to function parameters as long as to the return value.
In this case, the method is private and we wouldn't break any backward compatibility. What do you think?